### ISSUE TITLE
Nukie Reinforcements

### ISSUE THREAD

**Issue by TheOmn1** [HippieStation/HippieStationdeprecated2020#11987]
Currently the admins stance is when WAR is declared, no one is allowed to ghost or suicide making reinforcements bought effectively useless if there is even a chance for a ghost to join in at all.

I attempted to just save TCs and kill people on the station so ghosts can be brought in as borgs as I went but that doesn't work either because reinforcements can only be used on the syndie station.

I hope its possible to change it so borgs at least, if not ops, can be bought while on station so fresh ghosts can be used as resources.

**Comment by Manly-Dwarf** [comment]
alternatively you could have reinforcements ping ghosts, and if that fails, spawn a reinforcement spawner like a ghost role.

**Comment by ghost** [comment]
No, if u keep a lot of TCs (especially when war is declared), u can make spawn a shitton of syndieborg on the station and literally murderboning everyone, it would be completely overabused by nukie by spamming the bots.

**Comment by ghost** [comment]
@PortalVector I don't think that's how it works. If they use TCs for borgs, they won't be using them for other things (like mechs and guns). You also have to kill crewmembers to spawn this 'shitton', which would be quite difficult if it's warops - you have barely any equipment and the entire crew would stun rush you as soon as they got even a single rumour of your location.

**Comment by ghost** [comment]
@HideAndSeekLOGIC In some ways, it's completely true but i mean cyborgs are op, i suggest to only spawn "crew-based cyborg" or remove at least the Grenade Launcher i really hate that during war-ops round, i'm not okay with the fact to have 3 assault syndie rushing you.

**Comment by ghost** [comment]
A lot of warops things are OP. It's the balancing factor of three idiots vs. a crew of autists.

Plus, if warops is declared all you really need to do is fill a box with flashes and always keep one in your spare hand. See a borg? Run right at them while spamming Z.

**Comment by TheOmn1** [comment]
> No, if u keep a lot of TCs (especially when war is declared), u can make spawn a shitton of syndieborg on the station and literally murderboning everyone, it would be completely overabused by nukie by spamming the bots.

The big issue is that a significant amount of the toys available to warops cannot be used because of admins preventing them for some bizzare reason. And the only way to then use them based on the limitations of the admins (a.k.a. making some ghosts yourself) can't be used because the game doesn't accomodate them. It becomes a dead option if nothing else.

Also considering war declared means that the crew can actually get prepared and a very competent crew gets very well prepared (e.g. science tech shells which can actually shoot ion given out to the whole crew for example) spamming bots is hardly OP either.

**Comment by TheOmn1** [comment]
Also I think another option to remedy the issue is to allow non essential jobs the option to be part of the reinforcements as well. Those people like greys aren't going to play/contribute anyway so may as well put them into the pool.

**Comment by Majick1** [comment]
Getting reinforcements isn't actually a problem because at normal pop 2-3 people are gonna see they aren't antag and suicide anyway.

**Comment by TheOmn1** [comment]
> Getting reinforcements isn't actually a problem because at normal pop 2-3 people are gonna see they aren't antag and suicide anyway.

Considering admins are against people suiciding when war is declared. I doubt it. This is an artificial scarcity caused by the admins and the one obvious way of remedying it isn't allowed by the code either so yes, reinforcements is still a huge problem and is broken as it.

**Comment by ghost** [comment]
Perhaps, if war was declared, it should also prompt low-priority jobs, such as Assistants.

**Comment by Majick1** [comment]
That rule is literally unenforced, it happens so often there's a running joke about one player suiciding/killbaiting to roll mid round and nothing has been done.

Also on the assistant thing, we should be doing more to encourage experienced players to play actual department roles, this has the opposite effect
